src/RpcClient/Utility.cs

```diff
@@ -253,9 +253,18 @@ public static StackItem StackItemFromJson(JObject json)
                 case StackItemType.Pointer:
                     return new Pointer(null, (int)json["value"].AsNumber());
                 case StackItemType.InteropInterface:
-                    return new InteropInterface(new object());
+                    return new InteropInterface((json.ContainsProperty("id") ? json["id"]?.AsString() : string.Empty) ?? string.Empty);
```

Comment:
I think we shouldn't use `string` as the inner type. We should create a new type. Otherwise we cannot extend it in the future.

Reply:
Use JObject as inner type directly？